app/src/internal/java/com/kickstarter/ui/activities/FeatureFlagsActivity.kt

```diff
@@ -1,23 +1,17 @@
 package com.kickstarter.ui.activities
```

Comment:
Tested on my environment, This activity looks good to me
<img width="469" alt="Screen Shot 2021-08-06 at 3 41 40 PM" src="https://user-images.githubusercontent.com/4083656/128577682-3ebba9f6-83ef-45e2-b36f-cec35b076c01.png">

Reply:
😍